rules/falco-incubating_rules.yaml

```
@@ -769,7 +769,7 @@
 # https://github.com/draios/sysdig/issues/954). So in that case, allow
 # a setuid.
 - macro: known_user_in_container
-  condition: (container and user.name != "N/A")
+  condition: (container and user.name != <NA>)
```

possible alternatives
```suggestion
  condition: (container and not user.name in ("<NA>","N/A"))
```